python/lsst/pipe/tasks/postprocess.py

```diff
@@ -555,6 +551,8 @@ def run(self, parq, funcs=None, dataId=None):
             Functors to apply to the table's columns
         dataId : dict, optional
             Used to add a `patchId` column to the output dataframe.
+        band : `str`, optional
+            Filter band that is being processed.
```

Comment:
Is this ever passed to `run`? `runDataRef` doesn't use it, and it looks like this task doesn't support Gen 3 yet.